packages/big-design/src/components/Table/Item/Item.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { memo, useContext } from 'react';
```

Comment:
Why did you end up changing the name to `Item` instead of `Row`? `Row`, to me at least, is more verbose of what you are actually using in a `Table` component. `Item` could be anything.

Reply:
I changed it since we accept an array of `Items`, so it made sense to me to have an `Item` component,  I don't mind changing it to back `Row` tho (just the component, not the prop).

Reply:
I feel like we should change it over to `Row` internally.